packages/opencensus-instrumentation-ioredis/src/ioredis.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {BasePlugin, CanonicalCode, Span, SpanKind} from '@opencensus/core';
+import * as ioredis from 'ioredis';
+import * as semver from 'semver';
+import * as shimmer from 'shimmer';
+
+export type IORedisCommand = {
+  reject: (err: Error) => void|undefined;
```

Comment:
Nit, I think returning `undefined` is essentially the same as `void`, so what would you think about justing using `=> void` here and for `resolve` below?